lib/screens/v2/candidate_generator/gl_eink.ex

```diff
@@ -0,0 +1,90 @@
+defmodule Screens.V2.CandidateGenerator.GlEink do
+  @moduledoc false
+
+  alias Screens.Config.Screen
+  alias Screens.Config.V2.{Footer, GlEink, Header}
+  alias Screens.V2.CandidateGenerator
+  alias Screens.V2.WidgetInstance.{FareInfoFooter, NormalHeader, Placeholder}
+
+  @behaviour CandidateGenerator
+
+  @impl CandidateGenerator
+  def screen_template do
+    {:screen,
+     %{
+       normal: [
+         :header,
+         :main_content,
+         :medium_flex,
+         :footer
+       ],
+       bottom_takeover: [
+         :header,
+         :main_content,
+         :bottom_screen
```

Comment:
Another thought: as we build these templates out, I'm wondering what the best way is to keep track of the slot IDs available in the templates and returned by the widgets' `slot_names`, in one place. For example, looks like this `:bottom_screen` slot only exists on the e-ink screen templates.

Reply:
We could use types for this purpose. I'm also not sure we need all of the different configurations that currently exist (for instance, we've never actually supported or used this `bottom_takeover` layout).